src/components/MaplibreMap.jsx

```diff
@@ -107,19 +107,19 @@ const MaplibreMap = ({ view, setView }) => {
   const prevDisplayedProjects = usePrevious(displayedProjects)
   const location = useLocation()
   const navigate = useNavigate()
-  const getURLParams = useCallback(() => {
+  const getURLParamsSnapshot = useCallback(() => {
     return new URLSearchParams(location.search)
   }, [location.search])
-  const queryParams = getURLParams()
-  const queryParamsLat = queryParams.get('lat')
-  const queryParamsLng = queryParams.get('lng')
-  const queryParamsZoom = queryParams.get('zoom')
-  const initialMapCenter = isValidLatLng(queryParamsLat, queryParamsLng)
-    ? [queryParamsLat, queryParamsLng]
+  const initialQueryParams = getURLParamsSnapshot()
```

Comment:
the name `queryParams` was used in block scope a couple of other places. One to declare a new var, another to update what is essentially initial param state which could have caused bugs